draft-ietf-jsonpath-base.md

```diff
@@ -958,19 +961,29 @@ In the resultant nodelist:
 
 Children of an object may occur in any order, since JSON objects are unordered.
 
-### Union Selector
+### List Selector
+
+The list selector allows combining member names, array indices, and
+slices in a single selector.
+
+Note: The list selector was called "union selector" in
+{{JSONPath-orig}}, as it was intended to solve use cases addressed by
+the union selector in XPath.
+However, the term "union" implies the set operation that involves
```

Comment:
If we keep this note (and I'm not sure we should), I suggest the following rewording:
```suggestion
However, the term "union" has the connotation of a set operation that involves
```